docs/content/concepts/dynamic-fields/transfers/transfer-to-object.mdx

```diff
@@ -191,3 +229,60 @@ module examples::account {
     }
 }
 ```
+
+### Soul-bound objects {#soul-bound-example}
```

Comment:
Can we promote this to a full-fledged guide/example (i.e. move it into `examples/move`, and create a document for it in the `guides` section that goes into a little more detail, and link to it from here?